views/component/course-events/template.njk

```diff
@@ -0,0 +1,19 @@
+{% if faceToFaceModules.length > 0 %}
+    <p class="govuk-body">Only modules which allow events are shown below.</p>
+{% else %}
+    <p class="govuk-body">No modules which allow events are part of the course.</p>
+{% endif %}
+
+{% for module in faceToFaceModules %}
+    <p class="govuk-!-font-weight-bold govuk-body">{{ module.title | default(course.title, true) }}</p>
+    {# TODO: style the anchor element  #}
```

Comment:
TODO - We try not to commit todos

Reply:
point taken - TODOs will be removed this week following further work on events